tests/baselines/reference/corrupted.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/compiler/corrupted.ts(1,1): error TS1127: Invalid character.
+
+
+==== tests/cases/compiler/corrupted.ts (1 errors) ====
+    ���
+    
+!!! error TS1127: Invalid character.
```

Comment:
How do you feel about:

> This file appears to contain invalid text and cannot be parsed.

Reply:
Yeah, if we're not going to backport then I'll make a new diagnostic